src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -480,7 +477,8 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 				overlay.overlayTarget = getRandomItem(random, ElementUtil.loadScreens());
 			}
 			return overlay;
-		case GUI:
+		} else if (ModElementType.GUI.equals(modElement.getType())) {
+			ArrayList<GUIComponent> components;

```

Comment:
Put this back, no reason to split it like this.

Reply:
Split what? Yesterday, you said it was good.
![image](https://user-images.githubusercontent.com/38427877/125517790-69ec2500-c220-49ad-9059-d20b2d14a614.png)

Reply:
if is ok, but no need to split array list like you did

Reply:
Oh, I didn't notice.